scripts/jenkins/qa_devstack.sh

```diff
@@ -242,4 +244,14 @@ tox -e smoke
 EOF
 fi
 
+h_echo_header "Store tempest results"
+pip install junitxml
```

Comment:
I would install the python-junitxml rpm instead

Reply:
IIRC devstack install everything with pip, and installing with pip works out of the box.
I tried to install the rpm and is giving me dependencies errors, if possible I'd keep to continue using pip.

Reply:
:+1: